electricitymap/contrib/config/constants.py

```diff
@@ -1,5 +1,7 @@
 # Note: web/src/helpers.js contains some of the same information.
 
+EXCHANGE_FILENAME_ZONE_SEPARATOR = "→"
```

Comment:
Maybe this could just be "-" style dash, exchanges can go both ways so a directional arrow doesn't really fit. Also having a character that doesn't exist on a keyboard is a bit annoying.

Reply:
It can't be a dash as zone keys already have dashes in them. We could do underscore though and it was discussed before. 
The change is minimal so I can include it. I agree about the change. 

Aren't the config pieces you're mentioning only used in web?

Reply:
I'll let you update the web config pieces you were mentioning in a subsequent PR :)

Reply:
The underscore works well 👍